pkg/engine/handlers/validation/validate_assert.go

```diff
@@ -109,8 +112,28 @@ func (h validateAssertHandler) Process(
 	if err != nil {
 		return resource, handlers.WithError(rule, engineapi.Validation, "failed to apply assertion", err)
 	}
+	logger.V(2).Info("not nested errors", "errors", errs)
 	// compose a response
 	if len(errs) != 0 {
+		var allowExisitingViolations bool
+		if rule.Validation.AllowExistingViolations == nil {
+			allowExisitingViolations = true
+		} else {
+			allowExisitingViolations = *rule.Validation.AllowExistingViolations
+		}
```

Comment:
Would be good to create a method for the validation rule type to get `allowExistingViolations`.